tests/schema.test.ts

```diff
@@ -289,3 +289,46 @@ test("it does not expose pagination", async () => {
 
   expect(printSchema(schema)).toMatchSnapshot()
 })
+
+test('alias top-level input types only if not customized', async () => {
```

Comment:
```suggestion
test('renames top-level input types that are customized', async () => {
```

Or do I misunderstand?